src/01.UI-Elements/ListItem.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+ListItem.propTypes = {
+  titleListItem: PropTypes.string.isRequired,
+  isCheckmarked: PropTypes.bool,
+}
+
+export default function ListItem({ titleListItem, isCheckmarked }) {
+  return (
+    <ElementStyled>
+      <CheckboxStyled type="checkbox" checked={isCheckmarked} />
+      <TitleStyled>{titleListItem}</TitleStyled>
+    </ElementStyled>
+  )
+}
+
+const ElementStyled = styled.label`
+  margin-left: 1px;
+  display: flex;
+  align-items: center;
+
+  & :not(:last-of-type) {
+    margin-bottom: 12px;
+  }
+`
+const CheckboxStyled = styled.input`
+  margin-right: 10px;
+  transform: scale(1.5);
+`
+const TitleStyled = styled.span`
```

Comment:
If you want to avoid the <ElementStyled><CheckboxStyled>..<TitleStyled>.. naming structure in the JSX, you could just nest the styles in the sass way and keep, apart from the outer element, the normal naming structure (<input><span>)
Just easier to read in my opinion.